packages/ckeditor5-dev-build-tools/src/config.ts

```diff
@@ -0,0 +1,47 @@
+// TODO: When it's close to being production ready, remove the comment below.
```

Comment:
For now, this Rollup configuration file is only added for testing purposes. Review of it is optional.